lib/composer.lock

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
I am not sure of the added value of this file in our context, as we commit the libraries